### ISSUE TITLE
USB is not supported

### ISSUE THREAD

**Issue by jobsorrow** [Eddayy/flutter_star_prnt#19]
discover printers via 

```dart
List<PortInfo> list = await StarPrnt.portDiscovery(StarPortType.All);
```
is not include USB interface printer. The reason is the relevant piece of code is commented out

https://github.com/Eddayy/flutter_star_prnt/blob/f3006bcf26200e3b6d9bc221c817a26966758695/android/src/main/kotlin/io/eddayy/flutter_star_prnt/FlutterStarPrntPlugin.kt#L275-L279

When i uncomment it and retry again. It throw Invalid argument error. as official starPRNT SDK reference said

https://www.star-m.jp/products/s_print/sdk/starprnt_sdk/manual/android_java/en/api_stario_port.html#starioport-searchprinter
```
Parameter context Required when specifying USB: for target.
```

**Comment by evgk** [comment]
Any plans to merge this PR? Hit exactly the same issue.

**Comment by Eddayy** [comment]
merged and published as version 2.3.4
